.travis.yml

```diff
@@ -1,29 +1,27 @@
+sudo: false
 language: python
 env:
   - TOXENV=flake8-py27
   - TOXENV=flake8-py33
   - TOXENV=py26-dj14
   - TOXENV=py27-dj14
-  - TOXENV=py26-dj15
```

Comment:
Do we need to remove those ? I know they are no longer supported versions of django, but I suspect some developers are still using them.